terraform/cloudfoundry/variables.tf

```diff
@@ -0,0 +1,77 @@
+variable "region"     {
```

Comment:
It looks like some of this content duplicates and/or could be moved to `globals.tf`.

Does Terraform not complain about duplicate `variable` definitions?